packages/procps/top.c.patch

```diff
@@ -0,0 +1,52 @@
+--- procps-3.3.17.orig/top/top.c	2021-02-09 17:11:25.000000000 +0700
```

Comment:
Why copying https://github.com/vhqtvn/termux-packages/blob/for-merge-20210528/packages/procps/top-top.c.patch ?